docs/coordinator/api/remote.md

```diff
@@ -0,0 +1,68 @@
+# API
```

Comment:
Could you just add one or two more sentences somewhere here saying that basically this implements Prometheus's remote write API but also it can be used as a general purpose way to write data into m3coordinator/m3db even if you're not using Prometheus

Reply:
Will do.